ui/app/src/App.tsx

```diff
@@ -67,10 +64,10 @@ function App() {
           <ChartsThemeProvider themeName="perses" chartsTheme={chartsTheme}>
             <PluginRegistry getInstalledPlugins={getInstalledPlugins} importPluginModule={importPluginModule}>
               <LegacyDataSourceRegistry>
-                <QueryStringProvider queryString={searchParams} setQueryString={setSearchParams}>
+                <QueryStringProvider queryString={searchParams}>
                   <ErrorBoundary FallbackComponent={ErrorAlert}>
                     {/* temp fix to ensure dashboard refreshes when URL changes since setQueryString not reloading as expected  */}
-                    <ViewDashboard key={location.key} />
```

Comment:
I discussed this with @sjcobb  offline but we had to remove this because it was remounting the whole app and wiping out state. We will address the proper way of syncing URL state in a follow up PR.

Reply:
I'm working to fix this now so hopefully we'll be able to re-enable setQueryString before we publish v0.9!

Reply:
Yeah, I noticed this too locally. LMK if you need some help figuring it out.